outputs.tf

```diff
@@ -31,12 +31,47 @@ output "security_group_id" {
   value       = "${aws_security_group.main.id}"
 }
 
+output "subnet_group_arn" {
+  description = "The ARN of the db subnet group."
+  value       = "${aws_db_subnet_group.main.arn}"
+}
+
 output "subnet_group_id" {
   description = "The db subnet group name."
   value       = "${aws_db_subnet_group.main.id}"
 }
 
-output "subnet_group_arn" {
-  description = "The ARN of the db subnet group."
-  value       = "${aws_db_subnet_group.main.arn}"
+output "address" {
+  description = "The address of the RDS instance"
+  value       = "${aws_db_instance.main.address}"
+}
+
+output "arn" {
+  description = "The ARN of the RDS instance"
+  value       = "${aws_db_instance.main.arn}"
+}
+
+output "availability_zone" {
+  description = "The availability zone of the RDS instance"
+  value       = "${aws_db_instance.main.availability_zone}"
+}
+
+output "zone_id" {
+  description = "The canonical hosted zone ID of the DB instance (to be used in a Route 53 Alias record)"
+  value       = "${aws_db_instance.main.hosted_zone_id}"
+}
+
+output "resource_id" {
+  description = "The RDS Resource ID of this instance"
+  value       = "${aws_db_instance.main.resource_id}"
+}
+
+output "status" {
```

Comment:
Maybe we should also remove this variable, doesn't seem to be very deterministic and can change from run to run. (For example if a backup is in progress). https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.DBInstance.Status.html

Reply:
Yes, I wasn't really sure why we needed it before, but I guess it is pointless.